### PR TITLE
fix #277507: tour instructions for Mac incorrect

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Comment by anatoly-os** [comment]
PR created by not MacOS user ;)

**Comment by Jojo-Schmitz** [comment]
Yep, relying on the handbook to be correct

**Comment by MarcSabatella** [comment]
FWIW, I consider the tours that are currently provided as placeholders - I do plan to go through them all and potentially rewrite them, maybe delete a few, almost certainly add others, etc.  But probably not until after alpha 2, assuming we are targeting the next couple of days for that.

File: share/tours/welcome.tour [hunk 1]
```diff
@@ -36,6 +36,6 @@
     <Widget>palette-box</Widget>
     </Message>
   <Message>
-    <Text>That's the end! As you use MuseScore, more tours will pop up to further explain functionality. \nTo disable those, select "Do not show me tours" before closing this window. \nYou can also turn these off in Edit->Preferences</Text>
+    <Text>That's the end! As you use MuseScore, more tours will pop up to further explain functionality. \nTo disable those, select "Do not show me tours" before closing this window. \nYou can also turn these off in Edit->Preferences (Mac: MuseScore->Preferences)</Text>
```
[the remark this answers is not in the document]
Comment (anatoly-os) [reply share/tours/welcome.tour]: Damn, again reviewed 3 days ago and forgot to submit.

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply share/tours/welcome.tour]: done

**Pull request merged**